Week1/exercise_1.js

```diff
@@ -0,0 +1,76 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    multipleStatements: true
+});
+
+
+/*Connect to SQL server*/
+connection.connect(err => {
+    if (err) {
+        return console.error('Connection error: ' + err.stack);
+    }
+    console.log('Connected!');
+});
+
+
+/*SQL queries*/
+const createDatabaseAndTables =
+    `DROP DATABASE IF EXISTS meetup;
+    CREATE DATABASE meetup;
+    USE meetup;
+    
+    CREATE TABLE Invitee (
+    invitee_no INT AUTO_INCREMENT PRIMARY KEY,
+    invitee_name VARCHAR(100),
+    invited_by VARCHAR(100)
+    );
+    
+    CREATE TABLE Room (
+    room_no INT AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Small pet peeve, indentations are a bit off. Your indentations on `Meeting` are different than the rest